include/fizzy/fizzy.h

```diff
@@ -190,8 +204,8 @@ bool fizzy_find_exported_function(
 /// module, behaviour is undefined.
 FizzyInstance* fizzy_instantiate(const FizzyModule* module,
     const FizzyExternalFunction* imported_functions, size_t imported_functions_size,
-    const FizzyExternalTable* imported_table, const FizzyExternalGlobal* imported_globals,
-    size_t imported_globals_size);
+    const FizzyExternalTable* imported_table, const FizzyExternalMemory* imported_memory,
```

Comment:
The C++ API supports a vector of tables and memories, while the C API only supports a singular for both. While both can be at most one in wasm 1.0, there is a discrepancy between the two APIs.

Perhaps should also comment about this, as it may be confused to those familiar with the wasm spec.

In short: I think we should support a single table/memory in the C API as we can make a breaking API release if we do start supporting newer wasm versions. I would be inclined to say the same restriction would make sense in C++. Perhaps just open a tracking issue about this?

Reply:
#634

Reply:
I agree on having single table/memory.